### DISCUSSION TITLE
How can I use it on my mobile phone?

### DISCUSSION THREAD

**Discussion by Y1YU** [RafaelGB/obsidian-db-folder#406] [Q&A]
Hello, thanks for your database folder.

I'm a student in China. I surprisingly found it yesterday, but when I try to get it on my phone, it shows "No appropriate version found". So I want to know how I can get it on my phone. Thanks.

**Comment by RafaelGB** [comment]
Hi @Y1YU I moved your question to the discussion section.

Currently I am working on an stable version for mobiles but you can try it with the last version of the plugin (2.6.3) and Obsidian (0.15.9)
